webrender_traits/src/api.rs

```diff
@@ -248,6 +248,10 @@ impl RenderApi {
         self.api_sender.send(msg).unwrap();
     }
 
+    pub fn shut_down(&self) {
```

Comment:
It would be nice to have this called automatically when all the senders are gone, although I don't see an easy way to do that.